standalone/src/main/scala/filodb.standalone/FiloServer.scala

```diff
@@ -72,6 +75,11 @@ class FiloServer(watcher: Option[ActorRef]) extends FilodbClusterNode {
     }
   }
 
+  override def shutdown(): Unit = {
```

Comment:
Why is this needed?

Reply:
Same as above
> This is part of some trial and errors to get IngestionAndRecoverySpec multi-jvm spec to run.
I am finding that the ports are not being released in the restart step of the tests.